### ISSUE TITLE
seriesWebglBar stop increasing more bar

### ISSUE THREAD

**Issue by extrame** [d3fc/d3fc#1381]
I got the error:"Attribute buffer aYValue has incorrect number of elements 0.5, expected 1" when I use seriesWebglBar instead of seriesWebglLine on a growing image.

And I found the code
```
draw.y0Values = (...args) => {
    const builder = program.buffers().attribute(yValueAttrib);

    if (builder) {
        const existingYValues = builder.data();
        let yArray = new Float32Array(existingYValues.length);
        yArray = yArray.map((d, i) => [1, 2, 4].includes(i % verticesPerBar) ? args[0][Math.floor(i / verticesPerBar)] : existingYValues[i]);
        builder.data(yArray);
    } else {
        let yArray = new Float32Array(args[0].length * verticesPerBar);
        yArray = yArray.map((d, i) => [1, 2, 4].includes(i % verticesPerBar) ? args[0][Math.floor(i / verticesPerBar)] : d);
        program.buffers().attribute(yValueAttrib, attributeBuilder(yArray).components(1));
    }

    return draw;
};
```

which stop increasing the yArray when data.length is increasing because It reuse the existing yArray but in my case, it should use new data to build because its length is growing

---
edited to fix formatting

**Comment by chrisprice** [comment]
Thanks for the report @extrame. I believe this will be fixed by #1376.

**Comment by chrisprice** [comment]
Closing as I believe this is now fixed. If not, please comment.
